connect/values.yaml

```diff
@@ -16,7 +16,7 @@ connect:
     type: emptyDir
     values: {}
   imagePullPolicy: IfNotPresent
-  version: "1.0.0"
+  version:
```

Comment:
Do we not need the interpolated `.Chart.AppVersion` here to match what the Readme says?

Reply:
Yeah, that's cleaner indeed.